internal/operands/tekton-pipelines/reconcile.go

```diff
@@ -17,7 +17,7 @@ import (
 )
 
 // +kubebuilder:rbac:groups=tekton.dev,resources=pipelines,verbs=list;watch;create;update;delete
-// +kubebuilder:rbac:groups=*,resources=configmaps,verbs=list;watch;create;delete
+// +kubebuilder:rbac:groups=core,resources=configmaps,verbs=list;watch;create;delete
```

Comment:
Just out of curiosity, is there any difference between using `*` or `core`? AFAIK, it will refer to the same `apiGroup`, isn't it?

Reply:
Yes, `*` would grant permission to any API group in your cluster. Imagine that some custom API group, e.g. `kubevirt.io` would contain a custom ConfigMap resource that may be accessible (e.g. some ConfigMap that contains secrets).

I see that it's also possible to just specify an empty API group, as `""` (`groups=""`) and it would also refer to the core objects. Anyways, by using `core` we embrace least privilege principle and readability.

From the beginning I think that It was specified as `*` to speed up the development process (also some verbs) - because to have a specific value requires some effort.

Reply:
I love the explanation of that topic here [1]. Very nice guidance.

[1] https://learnk8s.io/rbac-kubernetes#modelling-access-to-resources